sdk/core/azure-core-v2/src/main/java/module-info.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This module contains the core classes for Azure SDK.
+ */
+module com.azure.core {
```

Comment:
We should use v2 here.
```suggestion
module com.azure.v2.core {
```